mapping.tex

```diff
@@ -43,15 +52,21 @@ \subsection{Mapping between SBOL 2 and SBOL 3}
-        \item If the \external{refinement} type is \external{useLocal}, then the \external{FunctionalComponent} referenced by the \external{local} attribute is replaced with a \sbol{SubComponent} and the \external{MapsTo} is replaced with a \sbol{ComponentReference}.  The \sbol{inChildOf} attribute of this \sbol{ComponentReference} references the object that has this \external{MapsTo} as a child.  The \sbol{hasFeature} attribute of this \sbol{ComponentReference} takes the value of the \external{remote} attribute from the \external{MapsTo} object.  Finally, a \sbol{Constraint} is added that has as the \sbol{subject} the new \sbol{SubComponent}, as the \sbol{object} the new \sbol{ComponentReference}, and the \sbol{restriction} type of \sbol{replaces}. 
+      \item For the \sbol{ComponentReference}, the \sbol{inChildOf} attribute of this \sbol{ComponentReference} attribute references the object that has the \external{MapsTo} as a child, and the \sbol{hasFeature} attribute references the object referred to by the \external{remote} attribute from the \external{MapsTo} object.
+      \item The \sbol{Constraint} links this \sbol{ComponentReference} and the \sbol{SubComponent} referred to be the \external{local} attribute from the \external{MapsTo} object.  The property values of the \sbol{Constraint} depend on the value of the \external{refinement} value for the \external{MapsTo} object:
+      	\begin{itemize}
+	\item If the refinement is \external{useRemote}, then the \sbol{restriction} is \external{replaces}, the \sbol{subject} is the \\ \sbol{ComponentReference} and the \sbol{object} is the \sbol{SubComponent}.
+	\item If the refinement is \external{useLocal}, then the \sbol{restriction} is \external{replaces}, the \sbol{subject} is the \sbol{SubComponent} and the \sbol{object} is the \sbol{ComponentReference}.
+	\item If the refinement is \external{verifyIdentical}, then the \sbol{restriction} is \external{verifyIdentical}, the \sbol{subject} is the \sbol{ComponentReference} and the \sbol{object} is the \sbol{SubComponent}.
+	\item The \external{merge} \external{refinement} was never well defined and rarely if ever used, so it has been removed from SBOL 3.x.  If a \external{merge} is encountered, it SHOULD be handled as a \external{useRemote}.      
         \end{itemize}
+     \item As an OPTIONAL optimization, if the \sbol{SubComponent} referred to be the \external{local} property of the \external{MapsTo} is a ``placeholder'' with no significant content apart from its \sbol{MapsTo} relationships, then it may be eliminated, all objects that pointed to it can point directly to the new \sbol{ComponentReference} instead, and all transitive constraints using it as a bridge reduced to link the endpoints directly. 
```

Comment:
to by the

Reply:
Also, the constraint can be removed too, right?

Reply:
I've pushed changes that address both of these.